appscale/tools/scripts/appscale.py

```diff
@@ -41,8 +41,26 @@ def main():
            "customize it for your particular cloud or cluster.", 'green')
     sys.exit(0)
   elif command == "up":
+    update_dir = ""
+    if len(sys.argv) > 2:
+      if sys.argv[2] != '--update':
+        cprint("Usage: appscale up [--update] <code directory to update>", 'red')
+        sys.exit(1)
+
+      if len(sys.argv) > 4:
+        dir_args = sys.argv[3:]
+        update_dir = ",".join(dir_args)
+
+      else:
```

Comment:
A single argument does not need to be a special case here. You could simplify with something like
```
if len(sys.argv) < 4:
  (print usage)

update_dir = sys.argv[3:]
```